TombEngine/Objects/TR4/Entity/tr4_guide.cpp

```diff
@@ -632,8 +634,8 @@ namespace TEN::Entities::TR4
 				item->Animation.TargetState = GUIDE_STATE_ACTIVATE_TRAP_CROUCHING;
 			else
 			{
-				if (item->Animation.AnimNumber != (object->animIndex + GUIDE_ANIM_IDLE_CROUCH) &&
-					item->Animation.FrameNumber == (GetAnimData(item).frameEnd - 20))
+				if (item->Animation.AnimNumber != GUIDE_ANIM_IDLE_CROUCH &&
+					item->Animation.FrameNumber == 20)
```

Comment:
Are we sure that `GetAnimData(item).frameEnd - 20` equals to `20`? To me, it should be `GetAnimData(item).EndFrameNumber - 20`. Proof in tomb4 code: https://github.com/Trxyebeep/tomb4-legacy/blob/a0620f455350aa4947bbcbe18b417b0368e69fad/TOMB4/game/guide.cpp#L486